### PR TITLE
docs: minor documentation updates and adjustments

### PR DESCRIPTION
**Pull request by AnnaShaleva**
The content of `docs`, `pkg/interop` and `pkg/rpc/client` is reviewed.

### CODE REVIEW

File: docs/notifications.md [hunk 1]
```diff
@@ -8,19 +8,21 @@ receive them as JSON-RPC notifications from the server.
 
 ## Events
 Currently supported events:
- * new block added
-   Contents: block.
+ * new block added<br>
+   Contents: block.<br>
```
Comment (roman-khimov) [inline docs/notifications.md:12]: Can we avoid this `<br>`? Format it all in a different way? It will improve "rendered" MD, but I'd like for it to be readable without rendering as well.

File: docs/conventions.md [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
 
-When submitting a new convention, please open an issue for discussion, if possible please highlight parts in the code where this convention could help the code readiblity or simplicity.
-
-
-## Avoid Named return paramters
+When submitting a new convention, please open an issue for discussion, if
+possible please highlight parts in the code where this convention could help the
+code readability or simplicity.
 
+## Avoid Named return parameters
```
Comment (roman-khimov) [inline docs/conventions.md:12]: s/Named/named/

File: docs/consensus.md [hunk 3]
```diff
@@ -67,8 +67,8 @@ systemd service file to run automatically on system startup.
 Notice that the default configuration has RPC and Prometheus services enabled,
 you can turn them off for security purposes or restrict access to them with a
 firewall. Carefuly review all other configuration options to see if they meet
-your expectations. Details on various configuration options (as well as CLI
-commands) are provided [here](cli.md).
+your expectations. Details on various configuration options are provided
+[here](node-configuration.md), CLI commands are provided [here](cli.md).
```
Comment (roman-khimov) [inline docs/consensus.md:71]: It's better to avoid links like `[here]`.

File: docs/compiler.md [hunk 4]
```diff
@@ -48,7 +50,7 @@ The compiler uses Go parser internally and depends on regular Go compiler
 presence, so make sure you have it installed and set up. On some distributions
 this requires you to set proper `GOROOT` environment variable, like
 ```
-export GOROOT=/usr/lib64/go/1.15
+export GOROOT=/usr/lib64/go/1.17
```
Comment (roman-khimov) [inline docs/compiler.md:53]: Doesn't really matter.

File: docs/cli.md [hunk 5]
```diff
@@ -35,220 +35,8 @@ If you want to use some non-default configuration directory path, specify
...
-| SaveStorageBatch | `bool` | `false` | Enables storage batch saving before every persist. It is similar to StorageDump plugin for C# node. |
-| SecondsPerBlock | `int` | `15` | Minimal time that should pass before next block is accepted. |
-| SeedList | `[]string` | [] | List of initial nodes addresses used to establish connectivity. |
-| StandbyCommittee | `[]string` | [] | List of public keys of standby committee validators are chosen from. |
-| StateRootInHeader | `bool` | `false` | Enables storing state root in block header. | Experimental protocol extension! |
-| StateSyncInterval | `int` | `40000` | The number of blocks between state heights available for MPT state data synchronization. | `P2PStateExchangeExtensions` should be enabled to use this setting.  |
-| ValidatorsCount | `int` | `0` | Number of validators. |
-| VerifyBlocks | `bool` | `false` | Denotes whether to verify received blocks. |
-| VerifyTransactions | `bool` | `false` | Denotes whether to verify transactions in received blocks. |
+Refer to the [node configuration documentation](./node-configuration.md) on
```
Comment (roman-khimov) [inline docs/cli.md:38]: This is an example of good link, btw.

Comment (roman-khimov) [reply docs/cli.md:38]: s/on/for/

File: docs/cli.md [hunk 6]
```diff
@@ -602,6 +395,10 @@ case), you can add `--gas` for extra network fee (raising priority of your
 transaction). And you can save transaction to file with `--out` instead of
 sending it to the network if it needs to be signed by multiple parties.
 
+To add optional `data` transfer parameter specify `data` positional arguments
```
Comment (roman-khimov) [inline docs/cli.md:398]: arguments? It's just one argument, I think.

File: docs/vm.md [hunk 7]
```diff
@@ -53,6 +52,7 @@ Commands:
   ops             Dump opcodes of the current loaded program
   parse           Parse provided argument and convert it into other possible formats
   run             Execute the current loaded script
+  slots           Show slots (static, local and arguments) contents
```
Comment (roman-khimov) [inline docs/vm.md:55]: Please move it to separate PR, we need something more like `slot X` here, dumping all slots at the same time is not very useful.

File: pkg/vm/slot.go [hunk 8]
```diff
@@ -1,6 +1,8 @@
 package vm
 
 import (
+	"encoding/json"
+
```
Comment (roman-khimov) [inline pkg/vm/slot.go:5]: beep

File: pkg/vm/context.go [hunk 9]
```diff
@@ -278,6 +279,21 @@ func (c *Context) IsDeployed() bool {
 	return c.NEF != nil
 }
 
+// Slot returns json formatted representation of the given slot.
+func (c *Context) Slot(n string) string {
```
Comment (roman-khimov) [inline pkg/vm/context.go:283]: This interface always felt strange to me (the same as the one for `Stack` in VM).

File: pkg/vm/slot.go (hunk 8 above)
Comment (AnnaShaleva) [reply pkg/vm/slot.go]: It is intentionally, otherwise, linter doesn't like it.

File: pkg/vm/context.go (hunk 9 above)
Comment (AnnaShaleva) [reply pkg/vm/context.go]: Will be fixed in https://github.com/nspcc-dev/neo-go/pull/2164.

**Review by roman-khimov — APPROVED** [review]
